src/entities/Profile.ts

```diff
@@ -5,7 +5,7 @@ import { IsString } from 'class-validator';
 export interface APIProfile {
 	id: string;
 	course: string;
-	yearOfStudy: number;
+	yearOfStudy: string;
```

Comment:
I think you can actually set this type to `Year` (not sure if anything will break, could you try switching and see?)